src/main/archetype/ui.tests.wdio/test-module/wdio.conf.commons.js

```diff
@@ -76,13 +75,7 @@ export const config = {
             showInBrowser: false,
             useOnAfterCommandForScreenshot: false,
             LOG: log4js.getLogger('default')
-        }],
-        ['video', {
-            saveAllVideos: false,
-            videoSlowdownMultiplier: 5,
-            videoRenderTimeout: 2,
-            outputDir: path.join('./reports', '/videos'),
-        }],
+        }]
```

Comment:
```suggestion
        }],
```

We normally always keep a trailing comma